examples/automm/object_detection/voc_inference.py

```diff
@@ -0,0 +1,28 @@
+import time
+
+from autogluon.multimodal import MultiModalPredictor
+from autogluon.multimodal.utils import from_voc
+
+
+def test_voc_inference(checkpoint_name="faster_rcnn_r50_fpn_2x_coco"):
```

Comment:
For VOC inference, it seems we are still using COCO trained model?

Reply:
Just updated the VOC inference, currently having S3 access issues. Will solve it soon.

Reply:
Done.